### PR TITLE
Implementing framework inspector mode. Fixes #9.

### LINKED ISSUE #9: Button to show visually which apps are written in each framework (colored border)
See https://github.com/CanopyTax/single-spa-examples/issues/7#issuecomment-260008143

### PR DESCRIPTION
**Pull request by internettrans**
See #9

### CODE REVIEW

File: src/angular1/base.template.html [hunk 1]
```diff
@@ -1,21 +1,26 @@
-<div class="container">
-	<div class="row">
-		<h4 class="light">
-			Angular 1 example
-		</h4>
-		<p class="caption">
-			This is a sample application written with Angular 1.5 and angular-ui-router.
-		</p>
+<div ng-style="vm.styles">
+	<div ng-if="vm.frameworkInspector">
```
Comment (internettrans) [inline src/angular1/base.template.html:2]: This is the new part

File: src/angular2/example-ng2-app.component.ts [hunk 2]
```diff
@@ -1,18 +1,44 @@
...
+	public showFramework:Boolean;
+
+	constructor(@Inject(NgZone) ngZone:NgZone) {
+		this.showFramework = false;
+		this.ngZone = ngZone;
+	}
+
+	ngOnInit() {
+		this.subscription = showFrameworkObservable.subscribe(showFramework => {
+			this.ngZone.run(() => {
```
Comment (internettrans) [inline src/angular2/example-ng2-app.component.ts:34]: $scope.$digest???????????????????

File: src/home/root.component.js [hunk 3]
```diff
@@ -1,76 +1,94 @@
...
+		this.state = {
+			frameworkInspector: false,
+		};
+	}
+	componentWillMount() {
+		this.subscription = showFrameworkObservable.subscribe(newValue => this.setState({frameworkInspector: newValue}));
+	}
+	render() {
+		return (
+			<div style={this.state.frameworkInspector ? {border: getBorder('react')} : {}}>
```
Comment (internettrans) [inline src/home/root.component.js:18]: This line is the new part

**Comment by internettrans** [comment]
@TheMcMurder this uses a ReplaySubject (which is a type of cold observable) to communicate between the apps.

**Comment by blittle** [comment]
:+1: love this!

**Comment by TheMcMurder** [comment]
I just saw this, this is awesome!

**Pull request merged**